.travis.yml

```diff
@@ -56,7 +56,6 @@ notifications:
           - http://julia.mit.edu:8000/travis-hook
 before_install:
     - make check-whitespace
-    - JULIA_SYSIMG_BUILD_FLAGS="--output-ji ../usr/lib/julia/sys.ji"
```

Comment:
Right before the release is not the time to remove tests for the no-linker use case because "at some point" we'll have a better solution. Jeff agreed several months ago, what's changed? Most of the makefile changes in that PR (with the exception of installing sys-all) are not as controversial as the CI/testing configuration changes. And they would allow exactly what I suggested above of building sys.ji separately (in parallel) on CI, in a way that doesn't need a linker, and testing that it can be used. A replacement for this test, rather than deletion, belongs in a rebased version of that PR, not this one.